chsdi/templates/htmlpopup/shop_product.mako

```diff
@@ -0,0 +1,31 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+
+<%
+   lang = lang if lang in ('fr','it','en') else 'de'
+   fid = c['featureId']
+   webDavHost = request.registry.settings['webdav_host']
+   scale_str = str(c['attributes']['scale'])
+   n = ""
+   while len(scale_str)>3:
+     scale_prov = int(scale_str)/1000
+     n = n + "'000"
+     scale_str = str(scale_prov)
+   scale = "1:" + scale_str + n
```

Comment:
you could create an helper function here.
`formatScale` sounds like a good name.
Provided an integer it would return a nicely formatted string.